README.md

```diff
@@ -80,29 +78,33 @@ The solution is to first build a shim static library with all the useful code vi
 - Python3:
 	You need to setup python3 on your Windows machine and install cffi (consider using a venv and pip)
 	
-1. Building the shim library
-	From the MSYS2 MINGW64 prompt, browse to the libbtc source and build it:
+**Building** 
+
+1. Building the shim library from the MSYS2 MINGW64 prompt, browse to the libbtc source and build it:
 	```
+	git clone https://github.com/libbtc/libbtc.git
+	cd libbtc
 	sh autogen.sh
 	./configure
 	make
 	```
-	Then browse to the c20p1305 source and build the shim lib:
+	Then, clone [c20p1305_cffi](https://github.com/goatpig/c20p1305_cffi.git) and browse to the c20p1305 source to build the shim lib:
 	```
-	mkdir build
-	cd build
-	cmake ..
+	git clone https://github.com/goatpig/c20p1305_cffi.git
+	cd c20p1305_cffi
+	mkdir build & cd build
+	cmake -G Ninja ..
 	ninja	
 	```
-2. Build the definition file
-	From Windows PowerShell, within the c20p1305/build/src folder:
+2. Build the definition file from Windows PowerShell, within the c20p1305/build/src folder:
+**Note**: *If MSVC was installed be sure to add lib.exe to your Windows path, otherwise this will fail.*
```

Comment:
"be sure to add lib.exe to the windows path after installing msvc". there is no lib.exe to be had without msvc to begin with